docs/contributing/core/index.md

```diff
@@ -73,67 +72,32 @@ On macOS, Developer Tools provides Clang for a C compiler.
 On Linux, [GNU Compiler Collection (GCC)](https://gcc.gnu.org/) is a common option.
 
 
-
+(contributing-core)=
 ## Install Plone core for development
 
 The tool that installs Plone core is `buildout.coredev`.
 
 The current default and development branch of `buildout.coredev` is `6.1`
-Older versions are named according to their `major.minor` version.
+Former versions are named according to their `major.minor` version.
 Its versions align with Plone's `major.minor` versions.
 
 Use a separate directory for each version of Plone to which you want to contribute.
-This will avoid switching between git branches, then re-running buildout, which can cause dependency conflicts between versions of Plone.
+This will avoid switching between git branches, which can cause dependency conflicts between versions of Plone.
```

Comment:
According to its [README](https://github.com/git/git/blob/master/README.md), "Git" is now capitalized. Originally it was "git", and occasionally "GIT", depending on your mood while using it, as the bulleted list in the README indicates.

```suggestion
This will avoid switching between Git branches, which can cause dependency conflicts between versions of Plone.
```